app/src/main/scala/Config.scala

```diff
@@ -0,0 +1,39 @@
+package lila.fishnet
+
+import cats.syntax.all.*
+import ciris.*
+import ciris.http4s.*
+import com.comcast.ip4s.*
+import cats.effect.IO
+
+object Config:
+
+  def load: IO[AppConfig] = appConfig.load[IO]
+
+  def appConfig = (
+    RedisConfig.config,
+    HttpServerConfig.config,
+    KamonConfig.config
+  ).parMapN(AppConfig.apply)
+
+case class AppConfig(redis: RedisConfig, server: HttpServerConfig, kamon: KamonConfig)
+
+case class HttpServerConfig(host: Host, port: Port)
+
+object HttpServerConfig:
+  def host   = env("HTTP_HOST").or(prop("http.host")).as[Host].default(ip"127.0.0.1")
```

Comment:
```suggestion
  def host   = env("HTTP_HOST").or(prop("http.host")).as[Host].default(ip"0.0.0.0")
```
would it be OK to bind to 0.0.0.0 by default like the previous .env value?

Reply:
thanks, also with this change, I think you can use for example `sbt app/run -Dredis.host=redis` to override config.